javascript.js

```diff
@@ -43,22 +51,18 @@ module.exports = {
           'sibling',
           'index'
         ],
-        'newlines-between': 'never'
+        'newlines-between': 'never',
+        alphabetize: {order: 'asc'}
```

Comment:
The only downside here is currently this IMO: https://github.com/benmosher/eslint-plugin-import/issues/1378#issuecomment-656323006. However I think it's still better that this is taken care of automatically, than us trying to keep the imports in order manually.